src/harness/BenchmarkDotNet.Extensions/BenchmarkDotNet.Extensions.csproj

```
@@ -6,8 +6,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="BenchmarkDotNet" Version="0.12.1.1528" />
-    <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.12.1.1528" />
+    <PackageReference Include="BenchmarkDotNet" Version="0.13" />
+    <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13" />
```

I am not sure if `0.13` is going to work. 

```suggestion
    <PackageReference Include="BenchmarkDotNet" Version="0.13.0" />
    <PackageReference Include="BenchmarkDotNet.Diagnostics.Windows" Version="0.13.0" />
```